xbmc/FileItem.cpp

```diff
@@ -1057,7 +1057,14 @@ bool CFileItem::IsPicture() const
   if (HasVideoInfoTag())
     return false;
 
-  return CUtil::IsPicture(m_strPath);
+  if (HasPVRTimerInfoTag() || HasPVRChannelInfoTag() || HasPVRChannelGroupMemberInfoTag() ||
+      HasPVRRecordingInfoTag() || HasEPGInfoTag() || HasEPGSearchFilter())
+    return false;
+
+  if (!m_strPath.empty())
+    return CUtil::IsPicture(m_strPath);
```

Comment:
Why not let CUtil::IsPicture() handle the empty case? Is it not expected to return false immediately if the path is empty?

Reply:
I wanted to avoid the additional function call.

Reply:
Fair.